pymchelper/readers/fluka.py

```diff
@@ -357,3 +361,38 @@ def get_unit_for_scoring(cls, scoring: str) -> str:
 
         # if unknown scoring, return empty string
         return ''
+
+
+@dataclass(frozen=True)
+class Axis:
+    """Single axis description"""
+
+    name: str
+    unit: str
+
+
+@dataclass(frozen=True)
+class AxesDescription:
+    """Axes descriptions"""
+
+    x: 'UsrbinAxes.Axis'
+    y: 'UsrbinAxes.Axis'
+    z: 'UsrbinAxes.Axis'
+
+
+class UsrbinAxes:
+    """Axes descriptions for USRBIN estimator"""
+
+    cartesian_mesh = AxesDescription(Axis("Position (X)", "cm"), Axis("Position (Y)", "cm"), Axis("Position (Z)", "cm"))
+    cylindrical_mesh = AxesDescription(Axis("Radius (R)", "cm"), Axis("Angle (PHI)", "rad"), Axis("Position (Z)", "cm"))
+    defaulf = cartesian_mesh
+
+    @classmethod
+    def get_axes_description(cls, binning_type: int) -> AxesDescription:
+        """Get axes descriptions for binding type"""
+        if binning_type in (0, 10):
+            return cls.cartesian_mesh
+        if binning_type in (1, 11):
+            return cls.cylindrical_mesh
+
+        return cls.default
```

Comment:
```suggestion
    def get_axes_description(binning_type: int) -> AxesDescription:
        """Get axes descriptions for binding type"""
        # by default (which includes binning_type 0..10 we have Cartesian mesh
        result = AxesDescription(Axis("Position (X)", "cm"), Axis("Position (Y)", "cm"), Axis("Position (Z)", "cm"))
        if binning_type in (1, 11):  # cylindrical mesh
            result AxesDescription(Axis("Radius (R)", "cm"), Axis("Angle (PHI)", "rad"), Axis("Position (Z)", "cm"))
        return result
```

Why not reducing this code to a simple function ? You can adjust the code as well